local-volume/helm/provisioner/templates/provisioner-cluster-role-binding.yaml

```diff
@@ -36,12 +36,12 @@ roleRef:
   kind: ClusterRole
   name: local-storage-provisioner-node-clusterrole
   apiGroup: rbac.authorization.k8s.io
-{{- end }}
+{{- if .Values.common.useJobForCleaning }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: jobsrole
+  name: local-storage-provisioner
```

Comment:
A more descriptive name could be "local-storage-provisioner-jobs-role"

Reply:
Done.